internal/util/K8sUtil.go

```diff
@@ -382,3 +382,16 @@ func (impl K8sUtil) DeleteAndCreateJob(content []byte, namespace string, cluster
 
 	return nil
 }
+
+func (impl K8sUtil) ListNamespaces(client *v12.CoreV1Client) (exists []v1.Namespace, err error) {
```

Comment:
we are not used named return values.. remove it and replace with return types only